### PR TITLE
Hue scene id with long touch gesture

### PR DESCRIPTION
**Pull request by commento**
Scene ids retrieved from nuimo configuration file (if present) and mapped on long touch nuimo gestures

### CODE REVIEW

File: senic_hub/nuimo_app/__init__.py [hunk 1]
```diff
@@ -125,6 +125,15 @@ def process_gesture(self, gesture, delta):
...
 
+        elif gesture == Gesture.LONGTOUCH_LEFT:
+            self.active_component.on_longtouch_left()
+
+        elif gesture == Gesture.LONGTOUCH_RIGHT:
+            self.active_component.on_longtouch_right()
+
+        elif gesture == Gesture.LONGTOUCH_BOTTOM:
+            self.active_component.on_longtouch_bottom()
+
```
Comment (larsblumberg) [inline senic_hub/nuimo_app/__init__.py]: Please change order to this: LEFT, BOTTOM, RIGHT.

This is because left is station 1, bottom is station 2 and right is station 3.

File: senic_hub/nuimo_app/components/__init__.py [hunk 2]
```diff
@@ -24,6 +24,15 @@ def start(self):
...
 
+    def on_longtouch_right(self):
+        pass
+
+    def on_longtouch_bottom(self):
+        pass
+
+    def on_longtouch_left(self):
+        pass
+
```
Comment (larsblumberg) [inline senic_hub/nuimo_app/components/__init__.py]: Please change order to this: LEFT, BOTTOM, RIGHT.

File: senic_hub/nuimo_app/components/light.py [hunk 3]
```diff
@@ -30,7 +30,7 @@ def update_from_ha_state(self, state):
         if received_brightness is not None:
             # Only overwrite our brightness if we haven't recently modified it ourselves
             # This avoids synching with an old brightness while the user is still turning the wheel
-            if time() - self.last_brightness_request_time > Component.BRIGHTNESS_CHANGE_RESPONSE_TIMEOUT:
+            if self.brightness is None or time() - self.last_brightness_request_time > Component.BRIGHTNESS_CHANGE_RESPONSE_TIMEOUT:
```
Comment (larsblumberg) [inline senic_hub/nuimo_app/components/light.py]: This shouldn't be part of this PR as it's already merged, let's rebase your branch to be on top of latest master commit

File: senic_hub/nuimo_app/components/philips_hue.py [hunk 4]
```diff
@@ -228,6 +228,21 @@ def __init__(self, config):
         self.lights = self.create_lights(config['lights'])
         self.lights.update_state()
 
+        if 'scene_id_1' in config:
+            self.scene_id_1 = config['scene_id_1']
+        else:
+            self.scene_id_1 = None
```
Comment (larsblumberg) [inline senic_hub/nuimo_app/components/philips_hue.py]: This and following can be simplified to:

    self.scene_id_1 = config.get('scene_id_1', None)

File: senic_hub/nuimo_app/components/philips_hue.py [hunk 5]
```diff
@@ -253,6 +268,21 @@ def filter_reachable(self, light_ids):
...
+    def on_longtouch_right(self):
+        logger.debug("on_longtouch_right()")
+        if self.scene_id_2 is not None:
+            self.bridge.activate_scene('0', self.scene_id_2)
+
+    def on_longtouch_bottom(self):
+        logger.debug("on_longtouch_bottom()")
+        if self.scene_id_3 is not None:
+            self.bridge.activate_scene('0', self.scene_id_3)
+
```
Comment (larsblumberg) [inline senic_hub/nuimo_app/components/philips_hue.py]: Please change order to: left, bottom, right and adopt scene numbering as committed above

Comment (larsblumberg) [inline senic_hub/nuimo_app/components/philips_hue.py]: Each long touch should show a station matrix, you can use these:

        "         " +
        "     *   " +
        "    **   " +
        "   * *   " +
        "     *   " +
        "     *   " +
        "     *   " +
        "     *   " +
        "         "

        "         " +
        "   ***   " +
        "      *  " +
        "      *  " +
        "   ***   " +
        "  *      " +
        "  *      " +
        "   ***   " +
        "         "

        "         " +
        "   ***   " +
        "      *  " +
        "      *  " +
        "    **   " +
        "      *  " +
        "      *  " +
        "   ***   " +
        "         "

File: senic_hub/nuimo_app/components/philips_hue.py (hunk 4 above)
Comment (larsblumberg) [inline senic_hub/nuimo_app/components/philips_hue.py]: Please rename the config parameters to `station1`, `station2`, `station3`

File: senic_hub/nuimo_app/components/light.py (hunk 3 above)
Comment (commento) [inline senic_hub/nuimo_app/components/light.py]: ok I will sync the next time before the pull request
